pep-0514.txt

```diff
@@ -187,6 +226,7 @@ may look like this::
         SupportUrl = "http://www.example.com/distro-3"
         Version = "3.0.12345.0"
         SysVersion = "3.6.0"
+        SysArchitecture = "64bit"
 
```

Comment:
Alternatively, a virtualenv user might be able to provide the index into a named list. But if all virtualenv needs is the path and the user is expected to figure that out on their own, that's also fairly hostile.

(Or virtualenv could display a list of all known paths to installed Python interpreters, which would not be reliable without this PEP, regardless of the format of the IDs)